packages/utils/interactive-hooks/src/Pressability/Pressability.types.ts

```diff
@@ -1,60 +1,30 @@
 import { RectOrSize } from './InternalTypes';
 import { BlurEvent, FocusEvent, PressEvent, MouseEvent } from './CoreEventTypes';
-import { ViewProps } from 'react-native';
+import { ViewProps, PressableProps } from 'react-native';
 
-export type PressabilityConfig = Readonly<{
+export type PressablePressProps = {
   /**
-   * Whether a press gesture can be interrupted by a parent gesture such as a
-   * scroll event. Defaults to true.
-   */
-  cancelable?: boolean;
-
-  /**
-   * Whether to disable initialization of the press gesture.
-   */
-  disabled?: boolean;
-
-  /**
-   * Amount to extend the `VisualRect` by to create `HitRect`.
-   */
-  hitSlop?: ViewProps['hitSlop'];
-
-  /**
-   * Amount to extend the `HitRect` by to create `PressRect`.
-   */
-  pressRectOffset?: RectOrSize;
-
-  /**
-   * Whether to disable the systemm sound when `onPress` fires on Android.
-   **/
-  android_disableSound?: boolean;
-
-  /**
-   * Duration to wait after hover in before calling `onHoverIn`.
-   */
-  delayHoverIn?: number;
-
-  /**
-   * Duration to wait after hover out before calling `onHoverOut`.
+   * Called when a press gestute has been triggered.
    */
-  delayHoverOut?: number;
+  onPress?: PressableProps['onPress'];
 
   /**
-   * Duration (in addition to `delayPressIn`) after which a press gesture is
-   * considered a long press gesture. Defaults to 500 (milliseconds).
+   * Called when the press is activated to provide visual feedback.
    */
-  delayLongPress?: number;
+  onPressIn?: PressableProps['onPressIn'];
 
   /**
-   * Duration to wait after press down before calling `onPressIn`.
+   * Called when the press is deactivated to undo visual feedback.
    */
-  delayPressIn?: number;
+  onPressOut?: PressableProps['onPressOut'];
 
   /**
-   * Duration to wait after letting up before calling `onPressOut`.
+   * Called when a long press gesture has been triggered.
    */
-  delayPressOut?: number;
+  onLongPress?: PressableProps['onLongPress'];
+};
```

Comment:
Why do we still need these extra props? Are they not in core?

Reply:
Think of the stock Pressable component as a thin shim around the Pressability library in react-native. The code I brought in before in the /Pressability directory was a port of that code to our project (and to TypeScript). I'm mostly leaving this part unchanged for now. Also PressabilityConfig is not quite the same as PressableProps, they intersect but aren't equal.

The long term plan would be:

1. Get this change in
2. Switch all components that use useAsPressable() on View components to usePressableState on Pressable components.
3. Remove these old files along with the useAsPressable hook.

Reply:
The long term plan helps me understand the whole port of the Pressability library is going away, thank you. That explains to me why we still have these types.